docs/index.md

```diff
@@ -287,3 +231,68 @@ Individual resources must opt-in to providing configurable `timeouts`, and attem
 ~> **Note:** Terraform does not automatically rollback in the face of errors.
 Instead, your Terraform state file will be partially updated with
 any resources that successfully completed.
+
+## Migrating from the ProfitBricks provider
```

Comment:
Also, the migration guide can be moved to a different file, so only interested users (if there still are users who use profitbricks) would access it

Reply:
I actually removed it, I don't think there is anyone left using the profitbricks provider.